sources/packages/web/src/views/institution/locations/programs/LocationProgramView.vue

```diff
@@ -78,6 +97,18 @@
             header="Study Dates"
             :sortable="true"
           ></Column>
+          <Column field="validIntensity" header="Type" :sortable="true"
+            ><template #body="slotProps">
+              <span
+                v-if="slotProps.data.validIntensity === ValidIntensity.partTime"
```

Comment:
If the names of the types on DB are user-friendly please consider displaying them directly here.